tools.go

```diff
@@ -0,0 +1,5 @@
+// +build tools
+
+package tools
+
+import _ "github.com/gordonklaus/ineffassign"
```

Comment:
Would it make sense to remove this, but add something similar to #97? Could we slap an underscore import into //tools/tidy itself?

Reply:
That's also possible - the unbuildable tools.go is the conventional approach for this case, but we could converge it with #97. We'd probably have to merge #97 first before that's feasible.